### PR TITLE
Web config changes

### PR DESCRIPTION
**Pull request by manics**
First set of web docs changes for https://trello.com/c/ktgSRtvD/283-5-1-web-configuration

--no-rebase

### CODE REVIEW

File: omero/sysadmins/unix/install-web.txt [hunk 1]
```diff
@@ -241,30 +246,27 @@ Customizing your OMERO.web installation
...
 
-The default `fastcgi-tcp` option uses a TCP connection for communication
-between OMERO.web and your web server; `fastcgi` uses a file based socket and
-will require you to manage the file permissions on it. You will have to
-regenerate your web server configuration after changing this property
-(see :ref:`fastcgi_configuration`).
+OMERO.web is configured to use FastCGI TCP by default. If you are using a
+non-standard web server configuration and wish to use a file based socket
+you can set :property:`omero.web.application_server` to `fastcgi`, but you
+will have to create your own web server configuration file.
```
Comment (aleksandra-tarkowska) [inline omero/sysadmins/unix/install-web.txt]: I'd mention here "and will require you to manage the file permissions on it" and maybe kindly say that is hard?

**Comment by aleksandra-tarkowska** [comment]
@manics all looks good, how do you feel about the comment above?

**Comment by manics** [comment]
@aleksandra-tarkowska Updated, is this better?

**Comment by aleksandra-tarkowska** [comment]
looks good, thanks, good to merge

**Comment by hflynn** [comment]
Thanks, will merge once everything is green

**Comment by manics** [comment]
@hflynn Anything else here?

**Comment by hflynn** [comment]
Nope, merging now thanks!

**Pull request merged**